torch/_dynamo/external_utils.py

```diff
@@ -196,7 +197,18 @@ def get_nonrecursive_disable_wrapper(fn: Callable[_P, _R]) -> Callable[_P, _R]:
     # this function is in external_utils so that convert_frame doesn't skip it.
     @functools.wraps(fn)
     def nonrecursive_disable_wrapper(*args: _P.args, **kwargs: _P.kwargs) -> _R:
-        return fn(*args, **kwargs)
+        with fx_traceback.annotate(
+            {
```

Comment:
@malaybag since we decided to not support recursive=False, let's just throw a RuntimeError here if torch.compiler.is_exporting()?